deploy/docker/Dockerfile-cypress

```diff
@@ -0,0 +1,48 @@
+# don't forget to pass environmet variables to docker run like this:
+# podman run -e CYPRESS_BASE_URL=<kiali_url> -e CYPRESS_PASSWD=<passwd> docker-cypress:latest
+# more details here https://github.com/kiali/kiali/tree/master/frontend/cypress
+
+# using same baseimage of node as its defined in package.json
+FROM cypress/base:14.17.0
+
+# Run as non-root user for MP+/OpenShift
+USER node
+
+ENV KIALI_HOME=/opt/kiali \
+    PATH=$KIALI_HOME:$PATH
+
+WORKDIR $KIALI_HOME
+
+# "Fake" dbus address to prevent errors. (is this still needed?)
+# https://github.com/SeleniumHQ/docker-selenium/issues/87
```

Comment:
How is selenium involved?

Reply:
It's old fix for standalone chrome. Verified that it have no effects. Removing